docs/internals/release-process.txt

```diff
@@ -95,12 +138,12 @@ each version.
 
   * Major functionality bugs in newly-introduced features.
 
-  The rule of thumb is that fixes will be backported to the last major release
-  for bugs that would have prevented a release in the first place (release
-  blockers).
+  The rule of thumb is that fixes will be backported to the last feature
+  release for bugs that would have prevented a release in the first place
+  (release blockers).
 
 * Security fixes and data loss bugs will be applied to the current master, the
-  last two major releases, and the current :ref:`LTS release <lts-releases>`.
+  last two feature releases, and the current :term:`Long-term support release`.
```

Comment:
Actually, "current LTS release" is kind of mis-leading here, since there is a one-year overlap of LTS support. Really it should say "any LTS release still within its support window"